packages/react/__tests__/src/components/Pagination/index.js

```diff
@@ -28,29 +28,39 @@ describe('Pagination', () => {
           wrapper
             .find('button')
             .at(0)
-            .is('[aria-disabled="true"]')
-        ).toBe(true);
+            .getDOMNode()
+            .getAttribute('aria-disabled')
+        ).toBe('true');
```

Comment:
Enzyme struggles when you set `aria-disabled={true | false}` (boolean). Before, the TooltipTabstop was setting `aria-disabled="true"`.